app/models/discourse_single_sign_on.rb

```diff
@@ -189,14 +189,47 @@ def match_email_or_create_user(ip_address)
               override_gravatar: SiteSetting.sso_overrides_avatar
             )
           end
+          if profile_background_url.present?
+            Jobs.enqueue(:download_profile_background_from_url,
+              url: profile_background_url,
+              user_id: user.id,
+              is_card_background: false
+            )
+          end
+
+          if card_background_url.present?
+            Jobs.enqueue(:download_profile_background_from_url,
+              url: card_background_url,
+              user_id: user.id,
+              is_card_background: true
+            )
+          end
+
+          if profile_background_url.present?
```

Comment:
Why is this being repeated?

Reply:
I made this incorrect when I was rebasing the tree against master to create a clean commit history. The problem is already fixed currently.